common/lib/xmodule/xmodule/fields.py

```diff
@@ -252,3 +253,19 @@ def enforce_type(self, value):
             return value
 
         return self.from_json(value)
+
+
+class ScoreField(JSONField):
```

Comment:
Also, should this class have a `MUTABLE = False` declaration (it's important for performance, since we will otherwise do a defensive `deepcopy` of this field.